README.md

```diff
@@ -1 +1,20 @@
 # hw01
+
+My name is Mackenzie Whitfield.  
+I moved to Chicago from San Diego two weeks ago. I **love** the beach. My hobbies also include hiking, traveling, reading, and watching movies. Up until now, I have only lived in California.  
+
+# Where I have Lived 
+1. San Diego  
+2. San Francisco  
+3. Los Angeles
+4. Chicago
+
+# Education
+I studied psychology at Loyola Marymount University in Los Angeles. I began participating in undergraduate research my freshman year and completed an Honors Thesis in my senior year. One of the *most* memorable experiences I had as an undergraduate was my study abroad in Greece. Below is a link to the study abroad page specific to my program. If you go under Student Testimonials, you will see the one I wrote! 
+[LMU Odyssey Program](https://bellarmine.lmu.edu/moderngreek/beyondtheclassroom/studyabroad)
+
```

Comment:
This is a cool use of a link. I like how the text is different than the link address. It is a great way to provide more information about the topic that you are discussing, and it makes it easily accessible to the reader. I would love to see pictures of your trip!

```suggestion
 ![<alt text>](<insert path to image>)
```